### ISSUE TITLE
Comment has incorrect code  ( verbose=0. imbalanced=False [verbose=0, imbalanced=False])

### ISSUE THREAD

**Issue by GDGauravDutta** [AutoViML/featurewiz#102]
Actual :
import featurewiz as gwiz
wiz =gwiz.FeatureWiz(verbose=2)
Imported lazytransform v1.14. 

Imported featurewiz 0.5.4. Use the following syntax:
    >>> wiz = FeatureWiz(feature_engg = '', nrows=None, transform_target=True, scalers="std",
        		category_encoders="auto", add_missing=False, verbose=0. imbalanced=False,
        		ae_options={})
    >>> X_train_selected, y_train = wiz.fit_transform(X_train, y_train)
    >>> X_test_selected = wiz.transform(X_test)
    >>> selected_features = wiz.features
    
featurewiz is given 0.9 as correlation limit...
    Skipping feature engineering since no feature_engg input...
    final list of category encoders given: ['label', 'label']
    final list of scalers given: []

Expected :
import featurewiz as gwiz
wiz =gwiz.FeatureWiz(verbose=2)
Imported lazytransform v1.14. 

Imported featurewiz 0.5.4. Use the following syntax:
    >>> wiz = FeatureWiz(feature_engg = '', nrows=None, transform_target=True, scalers="std",
        		category_encoders="auto", add_missing=False, verbose=0**,** imbalanced=False,
        		ae_options={})
    >>> X_train_selected, y_train = wiz.fit_transform(X_train, y_train)
    >>> X_test_selected = wiz.transform(X_test)
    >>> selected_features = wiz.features
    
featurewiz is given 0.9 as correlation limit...
    Skipping feature engineering since no feature_engg input...
    final list of category encoders given: ['label', 'label']
    final list of scalers given: []

**Comment by AutoViML** [comment]
Yes this is fixed in the next version 0.5.5 
please upgrade via 
```pip install featurewiz --upgrade ```
thanks
